020-Getting-started/020-installation.mdx

```diff
@@ -91,6 +91,15 @@ client = XataClient()
 ```
 </TabbedCode>
 
+### Committing files
+
+To push and share your application on public repositories for others to use your project with their own Xata database, we recommend commiting the schema file generated with the command `xata schema dump -f schema.json`.
+Add instructions to your project's readme file for users to initiate a new database using the schema file `xata init --schema schema.json`. This command will also generate the client file `src/xata.ts` using the user's own database URL. In this case, the `.xatarc` file does not need to be committed.
```

Comment:
```suggestion
### Which files should be committed into my repo?

When working with others in either a public or private setting it's common to want to provide a way to replicate your local environment. If working on a specific project (rather than a template) in a team it is a good idea to commit the `.xatarrc` file containing the database information, the `src/xata.ts` file containing the client, and the `.xata` directory containing the migration files. Then, your team will only need to provide the `XATA_API_KEY` in their `.env` file to spin up your service.

For public templates or examples that are meant to be cloned, you wouldn't want to commit these files because they point to your specific database. Instead, it is recommended to provide the database schema in the repository by running `xata initi --schema schema.json`, and then provide instructions to recreate the database. This is often, but not always, done alongside providing CSV files. When working with templates, check the instructions on [this page](/docs/csv-data/import-data-cli) for ways to preload schemas and data programmatically.
```

Hope this is OK, but I rewrote this one almost completely. There's really two use cases, and I thought it might be better to describe why you'd want to do this stuff. The CSV page actually goes into good detail about ways to recreate these DBs so I think we might want to just point them there.

Reply:
Thanks for rewritting this cleanly @snide looks nice to me. I'll also wait for @joan-ing to cast an eye on it for copy.